src/preparers.py

```diff
@@ -532,3 +532,9 @@ def prepare_open_orca(row):
         {"from": "user", "text": inputs.strip(), "parent": row['source']},
         {"from": "assistant", "text": outputs.strip(), "parent": 0},
     ]
+
+def prepare_toxicchat(row):
+    return[
+        {"from": "user", "text": row["user_input"].strip(), "parent": "toxicchat0124", "score": float(1-row["toxicity"])},
+        {"from": "assistant", "text": row["model_output"].strip(), "parent": 0}
+    ]
```

Comment:
This looks like there are always 2 turns, but you put the format as `multi-turn dialog`? If it's always two turns then lets put `zero-shot` as format (I know its a weird name now, but we can change that for all datasets later).